proxy/src/serverless/error.rs

```diff
@@ -0,0 +1,5 @@
+use http::StatusCode;
+
+pub trait HttpCodeError {
```

Comment:
I wasn't sure if I should somehow scope it (as per https://doc.rust-lang.org/reference/visibility-and-privacy.html)